column_transforms/rasgo_drop_columns/rasgo_drop_columns.yaml

```diff
@@ -0,0 +1,23 @@
+name: rasgo_drop_columns
+transform_type: column
+source_code: rasgo_drop_columns.sql
+description: |
+  Drop columns by passing either an include_cols list of columns to include or an exclude_cols list of columns to exclude.
+  
+  Passing both include_cols and exclude_cols will result in an error.
+
+arguments:
+  include_cols:
+    type: column_list
+    description: A list of the columns from the dataset you want to keep.
+  exclude_cols:
+    type: column_list
+    description: A list of the columns from the dataset you want to drop. Any columns not in the exclude_cols list will be kept.
+example_code: |
+
+    t1 = source.transform(
+        transform_name='rasgo_drop_columns',
+        include_cols=["DS_WEATHER_ICON", "DS_DAILY_HIGH_TEMP", "DS_DAILY_LOW_TEMP"]
+    )
+
```

Comment:
For examples where it's ultra simple, do we want to showcase usage of all options? Like you could have here:
```
    t2 = source.transform(
        transform_name='rasgo_drop_columns',
        exclude_cols=["Some_column", "some other column"]
    )
```

Reply:
good idea - will add